### ISSUE TITLE
A11y_VirtualAgentBots_VirtualAgentHome_ChatDialog_Screenreader: Using narrator the chat appears from the bot is announced twice to the screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/BotFramework-WebChat#2442]
Environment Details:
Browser Version: Microsoft Edge 44.18362.267.0
Windows Version: Windows10
Screen reader: Narrator, NVDA (2019.2)
MAS Violated: MAS 1.3.1

**Steps to Reproduce:**
1. Home page should open for Virtual Agent Bots.
2. Navigate to main content area chat dialog,  the left side of the screen.
3. Start screen reader.
4. Navigate to the "Chat" section and then navigate to the "Type your message" edit field.
5. Interact with it and listen to how the chat is announced.

**Actual:**
When screen reader users navigate to the "Type your message" edit field and interact with it, the bot reply's to the user, the chat that appears on the screen is announced twice to the screen reader users which is very irritating to listen to for the users.

**This issue is observed only in Narrator and not in NVDA.**

**Expected:**
The chat that appears from the bot should be announced just once when screen reader users navigate to it.

**User Impact:**
If unnecessary information is announced to the screen reader users then the users will not understand the information that is conveyed to them which can lead to confusion.

**Comment by cwhitten** [comment]
Thank you the issue. However, we are in the midst of an internal a11y audit to meet WCAG 2.1. The mechanisms to open, track, and address issues uncovered during the audit is a different process. I will close this for now.
